frontend/public/components/resource-pages.ts

```diff
@@ -54,6 +55,8 @@ import {
 } from '../models';
 
 export const resourceDetailPages = ImmutableMap<GroupVersionKind | string, () => Promise<React.ComponentType<any>>>()
+  .set(AlertModelReference, () => import('./alert' /* webpackChunkName: "alert" */).then(m => m.AlertsDetailsPage))
```

Comment:
Do not add components here because they are for Kubernetes resources only.

Reply:
🎉